From 9c1ff85c7ea170bf4bfa6f0201c09d3522f792b5 Mon Sep 17 00:00:00 2001
From: Aihua Xu <aihuaxu@apache.org>
Date: Wed, 25 Jan 2017 09:25:17 -0500
Subject: [PATCH 0951/1164] CDH-49262 HIVE-15679: Some metastore metrics
 always contains column=null
 (write_column_statistics,
 write_partition_column_statistics) (Peter Vary,
 reviewed by Rajesh Balamohan)

(cherry picked from commit 608194fc5f6c6897c8860588d9dc51600e5f924f)

Change-Id: I58e7c806580bc2ec092ef7b4345ad3e460a0b118
---
 .../hadoop/hive/metastore/HiveMetaStore.java       |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/metastore/src/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java b/metastore/src/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java
index a4760c2..309428f 100644
--- a/metastore/src/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java
+++ b/metastore/src/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java
@@ -4650,7 +4650,7 @@ public boolean update_table_column_statistics(ColumnStatistics colStats)
       List<ColumnStatisticsObj> statsObjs =  colStats.getStatsObj();
 
       startFunction("write_column_statistics", ":  db=" + dbName
-          + " table=" + tableName + " column=" + colName);
+          + " table=" + tableName);
       for (ColumnStatisticsObj statsObj:statsObjs) {
         colName = statsObj.getColName().toLowerCase();
         statsObj.setColName(colName);
@@ -4697,7 +4697,7 @@ public boolean update_partition_column_statistics(ColumnStatistics colStats)
 
       startFunction("write_partition_column_statistics",
           ":  db=" + dbName + " table=" + tableName
-              + " part=" + partName + "column=" + colName);
+              + " part=" + partName);
       for (ColumnStatisticsObj statsObj:statsObjs) {
         colName = statsObj.getColName().toLowerCase();
         statsObj.setColName(colName);
-- 
1.7.9.5

